### ISSUE TITLE
Firefox for Android suggestions, issues

### ISSUE THREAD

**Issue by barcadams0** [mozilla-mobile/fenix#15187]
Desktop version of Firefox works pretty well. I use the Firefox for Android on my Samsung S8 device, I do not experience speed issues like others (there a lot of comments on Google Play about slow page loading and downloading concerns) but it would be useful if:
- Users could download full webpages as screenshot like in Desktop version, it subsitutes any pdf add-ons.
- If developers fix syncronization problems (sometimes I cannot see recently opened tabs from my PC)
-Mobile facebook page, which loads m.facebook.com by default on mobile devices, does not work perfectly: the images are unfocused,  the always loaded in low resolution. This problem is not exist in Google Chrome mobile app. 

I hope Firefox for Android does not stop developing.
Best Regards,
Ádám,  loyal Firefox User.

**Comment by hwinnemoe** [comment]
Please limit your feature requests/issue reports to _one_ single feature request/issue per GitHub issue, otherwise it is impossible for the developers to correctly tag the issues. Regarding your mentioned features/issues:
- See/follow #89 for the screenshot feature
- Sync issue - I do not understand, which direction does not work, do you open new tabs on your PC and they do not appear on your mobile device or the other direction, opened on mobile and then not visible on PC?
- See/follow BZ-1232091 for Facebook low resolution images

**Comment by barcadams0** [comment]
Dear hwinnemoe,

If I saw correctly threads, there are no solutions for these feature yet, maybe new update is going to add screenshot feauture and fix low res images on fb?
To answer your question: every synced page visible on Desktop, but on Firefox android, i do not always see my pages loaded on PC.

Regards,


Szalai Ádám

Geográfus
PhD hallgató

Szegedi Tudományegyetem
[cid:image001.jpg@01D5948F.387FEE40]


Feladó: hwinnemoe<mailto:notifications@github.com>
Elküldve: 2020. szeptember 18., péntek 13:08
Címzett: mozilla-mobile/fenix<mailto:fenix@noreply.github.com>
Másolatot kap: barcadams0<mailto:sgt.adams@hotmail.com>; Author<mailto:author@noreply.github.com>
Tárgy: Re: [mozilla-mobile/fenix] Firefox for Android suggestions, issues (#15187)


Please limit your feature requests/issue reports to one single feature request/issue per GitHub issue, otherwise it is impossible for the developers to correctly tag the issues. Regarding your mentioned features/issues:

  *   See/follow #89<https://github.com/mozilla-mobile/fenix/issues/89> for the screenshot feature
  *   Sync issue - I do not understand, which direction does not work, do you open new tabs on your PC and they do not appear on your mobile device or the other direction, opened on mobile and then not visible on PC?
  *   See/follow BZ-1232091<https://bugzilla.mozilla.org/show_bug.cgi?id=1232091> for Facebook low resolution images

**Comment by hwinnemoe** [comment]
- The screenshot feature #89 depends on BZ-1530967 and that does currently not have a high priority.
- Facebook's low resolution images is explained in more detail in this answer by Facebook https://bugzilla.mozilla.org/show_bug.cgi?id=1232091#c17
- Your sync issue seems to be a new issue, you might want to open a new GitHub issue for this and keep it limited to this specific issue.

**Comment by kbrosnan** [comment]
Closing this as there is more than one issue reported. You can file a separate issue only addressing the sync problem that would be helpful.
